lib/cylc/task_message.py

```diff
@@ -235,6 +238,12 @@ def _update_job_status_file(self, messages):
                     job_status_file.write("%s=%s\n" % (
                         self.CYLC_JOB_EXIT,
                         message.replace(self.FAIL_MESSAGE_PREFIX, "")))
+                elif message.startswith(self.ABORT_MESSAGE_PREFIX):
+                    job_status_file.write("%s=%s\n" % (
+                        self.CYLC_JOB_EXIT, "EXIT"))
+                    job_status_file.write("%s=%s\n" % (
+                        self.CYLC_JOB_EXIT_MESSAGE,
+                        message.replace(self.ABORT_MESSAGE_PREFIX, "")))
```

Comment:
Not sure if this is the right thing to do here? (new CYLC_JOB_EXIT_MESSAGE in status files, just for aborted jobs)